lib/interfaces/hapi.js

```diff
@@ -122,8 +122,8 @@ function hapiRegisterFunction(client, config, server, options, next) {
  */
 function handlerSetup(client, config) {
   var hapiPlugin = {register : hapiRegisterFunction.bind(null, client, config)};
-  var version = isPlainObject(config) && isString(config.version)
-                    ? config.version
+  var version = isPlainObject(config) && isString(config.version) ?
+                    config.version
```

Comment:
Since you're changing the style; preference would be to keep the ':' operator at the end of current line rather than at the start of the new line. In this case however, the two legs of the conditional are short, so you can both of them on the same line.